Holovibes/sources/core/icompute.cc

```diff
@@ -26,7 +26,8 @@ using camera::FrameDescriptor;
 void ICompute::fft_freqs()
 {
     uint time_transformation_size = setting<settings::TimeTransformationSize>();
-    float d = setting<settings::CameraFps>() / time_transformation_size;
+    float fs = setting<settings::CameraFps>();
+    float d = fs / time_transformation_size;
```

Comment:
Couldn't `fs` be replaced by putting `setting<settings::CameraFps>()` directly into the initialization of `d` ?